### PR TITLE
Updating the mhcquant functionalities for the non-quantified data

### PR DESCRIPTION
**Pull request by marissaDubbelaar**
Here, there is an update for the nf-core template (version 2.3.2) as well as a suggestion for the file creation for a run when `--skip_quantification` is provided. Of course, it is of interest to perform the quantification, but this is not always possible and it would be of interest to generate the belonging outcome.

What happened:
- nf-core template update (version 2.3.2)
- Adjustments of the `PRE_QUANTIFICATION` subworkflow: `OPENMS_FALSEDISCOVERYRATE`, `OPENMS_IDFILTER_FOR_ALIGNMENT`, and `OPENMS_TEXTEXPORTER_PSMS`
- Included `OPENMS_TEXTEXPORTER_UNQUANTIFIED`to write a combined FDR filtered output file for unquantified data
- Typo in previous release date

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/awsfulltest.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Launch workflow via tower
-        uses: nf-core/tower-action@v2
+        uses: nf-core/tower-action@v3
```
Comment (ggabernet) [inline .github/workflows/awsfulltest.yml:17]: In this file `pipeline: ...` and `revision: ....` should be removed. The change was introduced in the latest template update

File: .github/workflows/awstest.yml [hunk 2]
```diff
@@ -10,9 +10,9 @@ jobs:
     if: github.repository == 'nf-core/mhcquant'
     runs-on: ubuntu-latest
     steps:
+      # Launch workflow using Tower CLI tool action
```
Comment (ggabernet) [inline .github/workflows/awstest.yml:13]: Same here, `pipeline: ....` and `revision:....` are no longer needed.

File: .gitpod.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+image: nfcore/gitpod:latest
+
+vscode:
+  extensions: # based on nf-core.nf-core-extensionpack
+    - codezombiech.gitignore # Language support for .gitignore files
+    # - cssho.vscode-svgviewer              # SVG viewer
+    - davidanson.vscode-markdownlint # Markdown/CommonMark linting and style checking for Visual Studio Code
```
Comment (ggabernet) [inline .gitpod.yml:7]: There was a change in this line also in the last template update

**Review by ggabernet — COMMENTED** [review]
Hey @marissaDubbelaar , there was yet a new nf-core tools release and I think this is not yet updated to it

File: .github/workflows/awsfulltest.yml (hunk 1 above)
Comment (marissaDubbelaar) [reply .github/workflows/awsfulltest.yml:17]: Somehow I missed these lines

File: .github/workflows/awstest.yml (hunk 2 above)
Comment (marissaDubbelaar) [reply .github/workflows/awstest.yml:13]: Somehow I missed these lines, but it should be fixed now

File: .gitpod.yml (hunk 3 above)
Comment (marissaDubbelaar) [reply .gitpod.yml]: Adjusted this to the latest update

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,22 @@
 # nf-core/mhcquant: Changelog
 
-## v2.2.0 nfcore/mhcquant "Silver Titanium Ostrich" - 2021/01/14
+## v2.3.0 nfcore/mhcquant "White Gold Swallow" - 2022/mm/dependency
+
+### `Added`
+
+- nf-core template update (version 2.3.1)
```
Comment (ggabernet) [inline CHANGELOG.md:7]:
```suggestion
- nf-core template update (version 2.3.2)
```

File: README.md [hunk 5]
```diff
@@ -37,50 +37,41 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
-                                  --predict_class_1
-                                  --refine_fdr_on_predicted_subset
-    ```
+   ```bash
+   nextflow run nf-core/mhcquant -profile test,<docker/singularity/podman/shifter/charliecloud/conda/institute>
+                                 --input 'samples.tsv'
+                                 --fasta 'SWISSPROT_2020.fasta'
+                                 --allele_sheet 'alleles.tsv'
+                                 --predict_class_1
+                                 --refine_fdr_on_predicted_subset
```
Comment (ggabernet) [inline README.md:63]:
```suggestion
                                 --refine_fdr_on_predicted_subset
                                 --outdir ./results
```

Comment (ggabernet) [reply README.md:63]: Now outdir is compulsory

File: docs/usage.md [hunk 6]
```diff
@@ -90,7 +71,7 @@ Note that the pipeline will create the following files in your working directory
 
 ```console
 work            # Directory containing the nextflow working files
-results         # Finished results (configurable, see below)
+<OUTIDR>        # Finished results in specified location (defined with --outdir)
```
Comment (ggabernet) [inline docs/usage.md:74]:
```suggestion
<OUTDIR>        # Finished results in specified location (defined with --outdir)
```

**Pull request merged**